microsoftgraph/microsoft-graph-explorer-v4
# Feature: list all permissions

## Overview

Adds the ability to see all the delegated permissions available to a GE user.
A sliding panel is available behind the gear icon for logged in users and gives a user the opportunity to consent to multiple permissions at a go.

Fixes #471 and [AB#4427](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/4427)

### Demo

How to activate:
![image](https://user-images.githubusercontent.com/58787602/79737439-fc583080-8303-11ea-8610-7768bfca152e.png)


Permissions display
![image](https://user-images.githubusercontent.com/58787602/79737344-dd599e80-8303-11ea-8724-d4bb624c5e60.png)




### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Log in to graph explorer
*Click the gear icon
* Notice the view all permssions option
* Clicking on it opens a sliding panel that has a list of all permissions with the ones you have not consented to having a checkbox to choose .
* Check the checkbox, click modify permissions and notice they all change to consented
